pandas/tests/series/test_validate.py

```diff
@@ -0,0 +1,32 @@
+from pandas.types.validate import _validate_bool_type
+from unittest import TestCase
+from pandas.core.series import Series
+
+class TestSeriesValidate(TestCase):
+
+    s = Series([1,2,3,4,5])
```

Comment:
missing spaces after commas (PEP8). You can see on travis what is all failing: https://travis-ci.org/pandas-dev/pandas/jobs/181229873 (scroll down, at the bottom there is a list of failures)